data/content/assets/glyph-caret-right.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="5" height="24" viewBox="0 0 5 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M0.247788 8.20425C0.560345 7.91574 1.04761 7.93523 1.33612 8.24779L4.7997 12L1.33612 15.7522C1.04761 16.0648 0.560345 16.0843 0.247788 15.7957C-0.0647694 15.5072 -0.08426 15.02 0.204254 14.7074L2.7034 12L0.204254 9.29259C-0.08426 8.98003 -0.0647694 8.49277 0.247788 8.20425Z" fill="#0C0C0D"/>
+</svg>
```

Comment:
Running it by node module svgo will give an optimized version of this svg.

Reply:
Done!